esp32/README.md

```diff
@@ -0,0 +1,30 @@
+# esp32
+
+[![Crates.io](https://img.shields.io/crates/v/esp32?labelColor=1C2C2E&color=C96329&logo=Rust&style=flat-square)](https://crates.io/crates/esp32)
+[![docs.rs](https://img.shields.io/docsrs/esp32?labelColor=1C2C2E&color=C96329&logo=rust&style=flat-square)](https://docs.rs/esp32)
+![Crates.io](https://img.shields.io/crates/l/esp32?labelColor=1C2C2E&style=flat-square)
+[![Matrix](https://img.shields.io/matrix/esp-rs:matrix.org?label=join%20matrix&labelColor=1C2C2E&color=BEC5C9&logo=matrix&style=flat-square)](https://matrix.to/#/#esp-rs:matrix.org)
+
+A **P**eripheral **A**ccess **C**rate (**PAC**) for the **ESP32** from Espressif. See the [svd2rust] repository for more infomation on how to use this crate.
```

Comment:
typo "infomation" -> "information" ... same in the other READMEs

Reply:
Oops, good eye!